static/js/comfortmodels.js

```diff
@@ -103,10 +103,10 @@ comf.test = function() {
     // reproduces the bug related to sweat saturation and heat loss from skin
     met_values = [1.7, 1.71, 1.72, 1.73, 1.74, 1.75, 1.76, 1.77, 1.78, 1.79, 1.8, 1.81, 1.82, 1.83, 1.84, 1.85, 1.86, 1.87, 1.88, 1.89, 1.9];
     for (var i = 0; i < met_values.length; i++){
-        console.log("MET:", met_values[i]);
+        // console.log("MET:", met_values[i]);
         var x = comf.pierceSET(34, 34, 4, 80, met_values[i], 0.4, 0); // not normal
         //var x = comf.pierceSET(34.08, 34.08, 4, 80, met_values[i], 0.4, 0) // normal
-        console.log(x)
+        // console.log(x)
```

Comment:
I think we should just remove all the commented console.log statements

Reply:
I agree, I will do that